src/field.rs

```diff
@@ -15,12 +15,83 @@ use std::ops::Deref;
 use std::os::raw::c_void;
 use tibrv_sys::*;
 
+pub enum DecodedField<'m> {
```

Comment:
While we're discussing style nits and `rustfmt`, if you prefer the lifetime annotations to be 'meaningful' i.e using `'m` or `'msg` instead of `'a` or `'b`, let's reformat all the lifetimes to be meaningful rather than just the new ones.

Reply:
I'm updating this PR to make the style consistent (`'a`); if we'd like to do the meaningful thing, let's keep that in a separate PR. Don't think it has much benefit here; we don't have a lot of confusing lifetimes flying around...

Once I've pushed that, I'm happy w/ this being merged as-is. I will add a couple of issues for other questions/comments in this thread for follow up.